src/main/java/seedu/address/logic/parser/SortCommandParser.java

```diff
@@ -20,7 +20,8 @@
 public class SortCommandParser implements Parser<SortCommand> {
 
     public static final String PARSE_EXCEPTION_MESSAGE = "Invalid syntax for sort. Please use the "
-            + "following syntax in []: sort [/byname][/byemail][/byphone][/byaddress] (Optional)[/reverse]";
+            + "following syntax: sort {/byattribute} {/reverse (optional)}. For example: sort /byname /reverse."
```

Comment:
Minor concern on phrasing. Maybe use "Example: smth' instead of "For example: smth'